fonts.css

```diff
@@ -0,0 +1,16 @@
+@font-face {
+  font-family: "GothamPro";
+  src: 
+      url("../fonts/GothamPro.woff") format("woff"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5542092)